modules/install-sno-single-node-openshift-requirements.adoc

```diff
@@ -0,0 +1,35 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_single_node/install-sno-single-node-ocp-on-bare-metal.adoc
+
+[id="single-node-openshift-requirements_{context}"]
+
+= Single node OpenShift requirements
```

Comment:
Other platforms, such as Azure and AWS use the pattern "Requirements for installing..."  If the purpose of this section is the same as those, please follow the pattern.